content/influxdb/v2.0/get-started.md

```diff
@@ -209,6 +209,10 @@ If you rename the binaries, all references to `influx` and `influxd` in this doc
 
 ### Install InfluxDB as a service with systemd
 
+{{% note %}}
+As of InfluxDB 2.0.3, `systemd` is required for Linux service support.
```

Comment:
This from @psteinbachs.

Reply:
I had @danxmoran double check this, and we still fall back to init in the deb/rpm packages.

Reply:
Ah! Okay. So let's delete this note, yes?

Reply:
Or edit it?

Reply:
I would just delete it for now, thanks.